src/components/navigation/header/Header.tsx

```diff
@@ -102,9 +101,16 @@ export const Header = ({
                     pathTranslations={pathTranslations}
                   />
                 )}
-                <Button size="large" type="secondary" background="light">
-                  <Text type="labelLarge">{t("contact_us")}</Text>
-                </Button>
+
+                {/* THIS SHOULD BE MOVED TO SANITY */}
```

Comment:
We could possibly use the contact email field under Admin --> Parent Company

Reply:
Should have been separate PR but fixed it here to sneak it in

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExOWltaG11Z2d6cWkwejJ0cmdwbmdkMHA4cHpjZ3RubWlrYTN1cHpvZiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/HPgKlUJEEPgP7Q5d4Y/giphy.gif)